### DISCUSSION TITLE
Is sequelize not releasing the sqlite file?

### DISCUSSION THREAD

**Discussion by Welfordian** [sequelize/sequelize#16244] [Help & Questions]
Hi all,

I am using Sequelize with Electron. Everything works fine, until I need to re-establish the connection. The documentation says I need to create a new instance of Sequelize once a connection has been closed, which I am doing.

```
Once sequelize.close() has been called, it's impossible to open a new connection. You will need to create a new Sequelize instance to access your database again.
```

The problem is, once I create a new instance, I receive the following error:-

`Error: SQLITE_NOTADB: file is not a database`

I _assume_ (yes, to assume is to make an ass out of u & me) this is happening because Sequelize still has a... lock(?) on the file.

Is there any way around this? I am trying to create a password manager application and the database is encrypted, but once the user puts in a correct password, the following password entries are "correct", regardless if they are *incorrect* because `sqlcipher` seems to keep the key in memory for the current connection.

Any ideas?

Thanks!

**Comment by WikiRik** [comment]
Can you try to set up a [SSCCE ](https://github.com/sequelize/sequelize-sscce) so we can check better what you are trying to do?
